### PR TITLE
Fix: Pan the map to correct location when clicking on search result not belonging to the current venue

### PR DESCRIPTION
**Pull request by ammapspeople**
# What

- Fix bug where (when using `kioskOriginLocationId`) clicking on a search result not belonging to the currently set venue would not pan the map to the clicked Location, but instead reset the map to the initial settings.

# How

- Made the `useMapBoundsDeterminer` hook react to changes in the "venue" prop instead of the internal `venueName` state. Also manually trigger map pan when using the Venue Selector, since the responsibility of doing that was previously mistakenly given to the `useMapBoundsDeterminer` hook.

### CODE REVIEW

File: packages/map-template/src/hooks/useMapBoundsDeterminer.js [hunk 1]
```diff
@@ -70,7 +70,7 @@ const useMapBoundsDeterminer = () => {
      */
     useEffect(() =>  {
         determineMapBounds();
-    }, [mapsIndoorsInstance, currentVenueName, venues, locationId, kioskOriginLocationId, pitch, bearing, startZoomLevel, categories]);
```
Comment (andreeaceachir142) [inline packages/map-template/src/hooks/useMapBoundsDeterminer.js:73]: We don't need the current venue here anymore?

Comment (ammapspeople) [reply packages/map-template/src/hooks/useMapBoundsDeterminer.js:73]: No, we only need the venue (the value of the venue prop that is externally exposed).

Comment (andreeaceachir142) [reply packages/map-template/src/hooks/useMapBoundsDeterminer.js:73]: Alright 😄

**Review by andreeaceachir142 — APPROVED** [review]
👏🏻

**Pull request merged**